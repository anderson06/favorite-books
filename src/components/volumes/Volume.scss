.fb-volume {
  cursor: pointer;
}

.fb-volume-title {
  margin-right: 20px;
}

.fb-thumbnail-icon {
  line-height: 42px;
}

.highlight {
  font-weight: bold;
}

.fb-volume-cover {
  position: absolute;
  width: 42px;
  overflow: hidden;
  left: 15px;
  display: inline-block;
  vertical-align: middle;
  border: 0;
}
