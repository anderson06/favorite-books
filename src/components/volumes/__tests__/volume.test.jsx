import React from 'react';
import Volume from '../volume';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Volume
      id="000"
      title="Fake title"
      description="Fake description"
      thumbnail="fake-thumbnail.png"
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
