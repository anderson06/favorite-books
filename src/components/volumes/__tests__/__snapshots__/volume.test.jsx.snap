// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="fb-volume collection-item avatar"
>
  <img
    className="circle"
    src="fake-thumbnail.png"
  />
  <span
    className="title"
  >
    Fake title
  </span>
  <p
    className="fb-truncate"
  >
    Fake description
  </p>
  <a
    className="secondary-content"
    href="#!"
  >
    <i
      className="material-icons"
    >
      grade
    </i>
  </a>
</li>
`;
